src/lib/utils/formatUptime.ts

```diff
@@ -4,5 +4,16 @@ export const formatUptime = (date: Date) => {
   const days = Math.floor(diff / (1000 * 60 * 60 * 24))
   const hours = Math.floor((diff % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60))
   const minutes = Math.floor((diff % (1000 * 60 * 60)) / (1000 * 60))
-  return `${days}d ${hours}h ${minutes}m`
+  
+  let uptime = ''
```

Comment:
👍 could you do the same change (and future ones) in harmony repo to so we keep the component in sync?

Reply:
sure!